src/cdk/menu/menu-trigger-base.ts

```diff
@@ -60,6 +60,9 @@ export abstract class CdkMenuTriggerBase implements OnDestroy {
   /** Template reference variable to the menu this trigger opens */
   menuTemplateRef: TemplateRef<unknown>;
 
+  /** Data to be passed along to the menu template */
```

Comment:
Optional nit: Should it be made clear that this will be available as the context in the menu template? Using the `context` terminology here to match with https://angular.io/api/core/ViewContainerRef#createembeddedview